tests/test_diffraction_objects.py

```diff
@@ -239,11 +240,29 @@ def test_dump(tmp_path, mocker):
     test.wavelength = 1.54
     test.name = "test"
     test.scat_quantity = "x-ray"
-    test.insert_scattering_quantity(
-        x, y, "q", metadata={"thing1": 1, "thing2": "thing2", "package_info": {"package2": "3.4.5"}}
-    )
-    with mocker.patch("importlib.metadata.version", return_value="3.3.0"), freeze_time("2012-01-14"):
+
+    with warnings.catch_warnings(record=True) as captured_warnings:
+        # Configure the warnings system to capture all warnings
+        warnings.simplefilter("always")
+
+        # Perform the method call that is expected to trigger the RuntimeWarning due to the specified wavelength
+        test.insert_scattering_quantity(
+            x, y, "q", metadata={"thing1": 1, "thing2": "thing2", "package_info": {"package2": "3.4.5"}}
+        )
+
```

Comment:
The code now captures the warning:

From:
<img width="563" alt="Screenshot 2024-09-19 at 7 26 55 AM" src="https://github.com/user-attachments/assets/a2085051-7d33-47bc-8ae7-b2ac8e6a78e6">

to: 

<img width="564" alt="Screenshot 2024-09-19 at 7 27 35 AM" src="https://github.com/user-attachments/assets/f8ef0c16-1a6a-49d4-8322-ef14c924fa11">